api/serverV2.go

```diff
@@ -65,6 +66,14 @@ func WithActionIndex() Option {
 	}
 }
 
+// WithWorkers is the option to manipulate api.LogsInRange's go routine numbers
+func WithWorkers() Option {
```

Comment:
After discussion with other engineers in our dev team, the `worker pool` pattern might be used in other package. So `config.API` isn't the good place to store `workerNumbers` param. So let's hardcode the param in the `coreservice.go` at first. It will be moved in the future. Sorry about the trouble.